packages/xo-server/src/api/metadata-backup.js

```diff
@@ -0,0 +1,102 @@
+export function createJob({ schedules, ...job }) {
+  job.userId = this.user.id
+  return this.createMetadataBackupJob(job, schedules)
+}
+
+createJob.permission = 'admin'
+createJob.params = {
+  name: {
+    type: 'string',
```

Comment:
Why is it not optional like `backupNg.createJob`?

Reply:
The name of the backup is required in the UI.

Reply:
I think it was a conscious choice of my part and the UI should not require it.

For now, simply update this PR to make it optional :slightly_smiling_face: